dashboard/app/models/script.rb

```diff
@@ -1929,11 +1930,14 @@ def get_feedback_for_section(section)
 
     script_levels.each do |script_level|
       next unless script_level.oldest_active_level.can_have_feedback?
+      current_level = script_level.oldest_active_level
```

Comment:
(very minor) Maybe swap this with the line above so we don't call `script_level.oldest_active_level` twice?

Reply:
Oh yeah good call